### DISCUSSION TITLE
New warning after upgrading

### DISCUSSION THREAD

**Discussion by eskewj** [LekoArts/gatsby-themes#1016] [Questions]
Hello, I recently upgraded gatsby from **2.23.20 to 4.24.1**.

Now I'm seeing this warning: 

> warn ./node_modules/@lekoarts/gatsby-theme-minimal-blog/src/@lekoarts/gatsby-theme-minimal-blog-core/components/homepage.tsx
> export 'Head' (reexported as 'Head') was not found in '../../../components/homepage' (possible exports: default)

Also, I lost IBM Plex Sans as my default font-family.  Not sure if that is related.

Any suggestions?  Thank you!

**Comment by LekoArts** [comment]
> Now I'm seeing this warning

Maybe you didn't update all packages or you have an old version of some child packages around. npm isn't always good at updating dependencies. As a band aid you could remove your `node_modules` folder and lock file and re-install again.

> Also, I lost IBM Plex Sans as my default font-family. Not sure if that is related.

This is one of the breaking changes I made for the new major versions of the themes: https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-minimal-blog/CHANGELOG.md#500

So if you update major versions you'll need to check the breaking changes and possible changes.

**Comment by eskewj** [comment]
I got my fonts sorted, thank you.

Still seeing the console warning though even after refreshing node_modules.  Just a warning, but I'm afraid it could affect SEO.
